libs/core-ui/src/lib/Cohort/ShiftCohort/ShiftCohort.tsx

```diff
@@ -14,19 +14,23 @@ import {
 } from "office-ui-fabric-react";
 import React from "react";
 
+import {
+  ModelAssessmentContext,
+  defaultModelAssessmentContext
+} from "../../Context/ModelAssessmentContext";
 import { CohortStats } from "../CohortStats/CohortStats";
 import { ErrorCohort } from "../ErrorCohort";
 
 export interface IShiftCohortProps {
   isOpen: boolean;
-  cohorts: ErrorCohort[];
   onDismiss: () => void;
   onApply: (selectedCohort: ErrorCohort) => void;
 }
 
 export interface IShiftCohortState {
   options: IDropdownOption[];
   selectedCohort: number;
+  savedCohorts: ErrorCohort[];
```

Comment:
is this fixing some bug with the cohorts in shift cohort?  not sure why this change is needed

Reply:
Previously we passed in a subset of all cohorts into this component (specifically the saved ones only). That adds logic in the layer above. Now that we have the context available we can just as well have that filtering in here. That's why this removes the cohorts from input and adds them to the state which is set internally. I did just notice that I have to move the context usage to its own method, though. Doing that now